testing/database.rst

```diff
@@ -74,8 +74,8 @@ it's easy to pass a mock object within a test::
 
     use AppBundle\Entity\Employee;
     use AppBundle\Salary\SalaryCalculator;
-    use Doctrine\Common\Persistence\ObjectManager;
     use Doctrine\Common\Persistence\ObjectRepository;
```

Comment:
This could be replaced too:
```suggestion
    use Doctrine\Persistence\ObjectRepository;
```

(that will involve reordering the lines though).

And maybe there are more occurrences of `Doctrine\Common\Persistence` here.